src/main/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceImpl.java

```diff
@@ -114,11 +114,15 @@ public HearingResponse saveHearingRequest(CreateHearingRequest createHearingRequ
     }
 
     @Override
-    public void updateHearingRequest(Long hearingId, UpdateHearingRequest hearingRequest) {
+    public HearingResponse updateHearingRequest(Long hearingId, UpdateHearingRequest hearingRequest) {
         validateHearingRequest(hearingRequest);
         validateHearingId(hearingId);
         validateVersionNumber(hearingId, hearingRequest.getRequestDetails().getVersionNumber());
         validateHearingStatusForUpdate(hearingId);
+
+        // TODO: What's the next Status?!
+        HearingEntity savedEntity = updateHearingStatusAndVersionNumber(hearingId, null, hearingRequest);
```

Comment:
updateHearingStatusAndVersionNumber(Long hearingId, String newStatus) can be called as no need to update the request

Reply:
Agreed